### DISCUSSION TITLE
Do not create dependency dashboard/close it if there are no issues

### DISCUSSION THREAD

**Discussion by dword-design** [renovatebot/renovate#29738] [Suggest an Idea]
### Tell us more.

I think it would be great if the dependency dashboard would only be created if there are actual issues, e.g. deprecated dependencies. Then it's clear that there is something to do, otherwise the user needs to open the issue to check. Also, the dashboard should be closed if the issues have been resolved.

**Comment by rarkins** [comment]
https://docs.renovatebot.com/configuration-options/#dependencydashboardautoclose
